lib/load.php

```diff
@@ -154,18 +154,21 @@ function gutenberg_is_experiment_enabled( $name ) {
 remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' ); // Turns off WordPress 6.0's stopgap handler.
 
 /*
- * If the Fonts Library is available, load the Font Face files, else load the Fonts API.
- * This strategy is temporary until the Fonts Library is merged. It's used here to allow
- * the Font Face (redesigned Fonts API) to be merged before the Fonts Library while
+ * If the Font Library is available, load the Font Face files, else load the Fonts API.
+ * This strategy is temporary until the Font Library is merged. It's used here to allow
+ * the Font Face (redesigned Fonts API) to be merged before the Font Library while
  * keeping Fonts API available for sites that are using it.
  */
-if ( defined( 'FONTS_LIBRARY_ENABLE' ) && FONTS_LIBRARY_ENABLE ) {
-	// Loads the Fonts Library.
-	require __DIR__ . '/experimental/fonts/fonts-library/class-wp-fonts-library.php';
-	require __DIR__ . '/experimental/fonts/fonts-library/class-wp-font-family-utils.php';
-	require __DIR__ . '/experimental/fonts/fonts-library/class-wp-font-family.php';
-	require __DIR__ . '/experimental/fonts/fonts-library/class-wp-rest-fonts-library-controller.php';
-	require __DIR__ . '/experimental/fonts/fonts-library/fonts-library.php';
+if (
+	( defined( 'FONT_LIBRARY_ENABLE' ) && FONT_LIBRARY_ENABLE ) ||
+	( defined( 'FONTS_LIBRARY_ENABLE' ) && FONTS_LIBRARY_ENABLE )
+) {
```

Comment:
@matiasbenedetto I left both constants in just in case someone is working on a PR or testing locally that is using the plural version.

This code is temporary. Once the frontend PR lands, then it can be removed.

So I think it's okay to leave it, i.e. just in case to help with developer workflow.

What do you think?

Reply:
Makes sense :+1: